parquet-avro/src/test/java/org/apache/parquet/avro/TestArrayCompatibility.java

```diff
@@ -653,8 +654,8 @@ public void testAvroCompatRequiredGroupInListWithSchema() throws Exception {
     Path test = writeDirect(
         "message AvroCompatRequiredGroupInListWithSchema {" +
             "  optional group locations (LIST) {" +
-            "    repeated group array {" +
-            "      optional group element {" +
+            "    repeated group bag {" +
+            "      optional group array_element {" +
```

Comment:
Thanks for the explanation, this makes more sense to me now. But why is there the "RequiredGroup" part in the test case name? I don't see any required groups in either the Parquet schema (written in string) or the Avro schema.